lib/grape-swagger/endpoint.rb

```diff
@@ -235,6 +235,11 @@ def parse_request_params(required)
       end
     end
 
+    def param_type_is_array?(param_type)
+      return false unless param_type
+      param_type == 'Array' || param_type =~ /\[\w+\]\z/
+    end
```

Comment:
so @LeFnord are you suggesting that the output for nested Arrays in formData would look like `address[line1]` as opposed to `address[][line1]` ?